src/r_exasol/connection/async_executor/async_executor_impl.h

```diff
@@ -24,6 +28,7 @@ namespace exa {
          * @param errorHandler Callback function to invoke if an error occurs.
          */
         void execute(exa::tBackgroundAsyncErrorFunction errorHandler) override {
+            ae_stack_trace_logger astl(__func__);
```

Comment:
Is there a reason why you don't use the macro STACK_PRINTER

Reply:
I didn't like the idea of getting this macro then in all other files, which include this header.

Reply:
ah, I see, the files which use the macro are cpp files and not headers. Yeah, that sounds reasonable.